bin/cron/build_contact_rollups_v2

```diff
@@ -21,4 +21,9 @@ ensure
   contact_rollups.report_results
 end
 
+def main
+  return false unless DCDO.get('contact_rollups_active', true)
```

Comment:
_tiny_ nit: I don't think you need the `false` here and can just say `return unless ...`